packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -322,8 +324,10 @@ $table-unsorted-icon-color: get-color-token-from-bg(
     /* stylelint-enable selector-class-pattern */
   }
 
+  tbody th,
   th,
   td {
+    background-color: #fff;
```

Comment:
**question:** Curious, why did you hard code white here?

Reply:
Apologies, I realize my question was confusing and didn't have enough context. My question was more about why we are hard-coding `#fff` here. It looks like the background color for the borderless `thead th` is defined with `$table-background-color` ([code reference](https://github.com/uswds/uswds/blob/develop/packages/uswds-core/src/styles/mixins/general/table.scss#L304)). Additionally, is it possible to just style all borderless `th` cells, rather than having separate definitions for `thead th` and one for `tbody th`?